src/quo/components/wallet/account_overview/style.cljs

```diff
@@ -98,7 +110,11 @@
 (defn icon-props
   [customization-color theme]
   {:color (colors/theme-colors
+           ;; deprecated warning suppressed but please rewrite if you have time
+           #_{:clj-kondo/ignore [:deprecated-var]}
```

Comment:
We have 86 TODOs in the code already) I don't think increasing this value by 100 with those deprecated suppression would make any change. Plus, we aren't really need rewriting what is deprecated. It would be nice, it would be better but it is not really a `TODO`. 

>Overall, I think things look good, though I wonder whether we should avoid suppressing warnings if we plan to do more PRs for resolving the warnings. 

Overall I'd prefer maintaining a zero-warnings policy (and ensuring it with the automatic PR check). To be sure that if anything triggered a warning in the future - that was seen and fixed. And warnings that we already have... well it is possible that we will fix them but it is also possible that not. In any case old warnings shouldn't stay in our way of preventing new ones. wdyt?